client/templates/admin/posts/posts.html

```diff
@@ -3,6 +3,33 @@
   {{ > postEditor shouldRenewModelOnUserSave=true }}
 
   {{ #if isReady }}
+
+    <!-- I hsd to add the menu here for now to make it place above the posts and underneath the editor -->
```

Comment:
@dagvonkr that was really bad. Check my commits on how this would be done without repetition.